modules/installation-configuration-parameters.adoc

```diff
@@ -1727,6 +1735,50 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`premium_LRS` or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`controlPlane.platform.azure.type`
+|Defines the azure instance type.
+|String
+
+|`controlPlane.platform.azure.zones`
+|The availability zones where the installation program creates control plane machines.
+|String list
+
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
+|`true` or `false`. The default is `false`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.name`
+|The name of the disk encryption set that contains the encryption key from the installation prerequisites.
+|String, for example `production_disk_encryption_set`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.resourceGroup`
+|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites. This resource group should be different from the resource group where you install the cluster to avoid deleting your Azure encryption key when the cluster is destroyed. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
+|String, for example `production_encryption_resource_group`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.subscriptionId`
+|Defines the Azure subscription of the disk encryption set where the disk encryption set resides. This secondary disk encryption set is used to encrypt compute machines.
+|String, in the format `00000000-0000-0000-0000-000000000000`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskSizeGB`
+|The Azure disk size for the VM.
+|Integer that represents the size of the disk in GB. The default is `128`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskType`
+|Defines the type of disk.
+|`standard_LRS`, `premium_LRS`, or `standardSSD_LRS`. The default is `premium_LRS`.
+
+|`platform.azure.defaultMachinePlatform.type`
+|The Azure machine type for control plane and compute machines.
+|The Azure machine type.
```

Comment:
How about changing `Azure machine type` to `Azure instance type` in line1771&line1772 to sync up with `compute.platform.azure.type` and `controlPlane.platform.azure.type`?